_config.yml

```diff
@@ -748,7 +763,6 @@ facebook_sdk:
 # This plugin depends on Facebook SDK.
```

Comment:
Carefully here. Facebook, like VK, have own SDK (API). I think need to make similar as VK.